rasa/core/policies/policy.py

```diff
@@ -123,6 +126,7 @@ def __init__(
         featurizer: Optional[TrackerFeaturizer] = None,
     ) -> None:
         """Constructs a new Policy object."""
+        self.config = config
```

Comment:
?
```suggestion
        self._config = config
```

Reply:
alright

Reply:
@twerkmeister Seems this is still open?

Reply:
Yes indeed, I didn't do it here because it would mean another 100+ line changes because it would ripple through all the policies. In Ted policy alone there are like 50 accesses to `self.config`. Let's do it in a focused, separate PR just after this to not clutter this one here?

Reply:
sounds good 👍🏻